vc-verifier/js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vc-verifier-js",
+  "version": "0.0.1",
```

Comment:
It's semver Major.Minor.Patch
Patch is for bug fixes
first version will be 0.1.0

Reply:
Changing it